doc/source/overview/high_level.md

```diff
@@ -0,0 +1,1084 @@
+|---------------------------------------------|-------|------------------------------------------|----------------|----------------------------|-------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| [Pertinent Positives](../methods/CEM.ipynb) | Local | Black-box, White-box (_TensorFlow_)      | Classification | Tabular (numerical), Image | Which set of features of a given instance is sufficient to ensure the prediction stays the same | [docs](../methods/CEM.ipynb), [paper](https://arxiv.org/abs/1802.07623) |
+
+Introduced by [Amit Dhurandhar, et al](https://arxiv.org/abs/1802.07623), a Pertinent Positive is the subset of features
+of an instance that still obtains the same classification as that instance. These differ from [anchors](anchors)
+primarily in the fact that they aren't constructed to maximize coverage. The method to create them is also substantially
+different. The rough idea is to define an **absence of a feature** and then perturb the instance to take away as much
+information as possible while still retaining the original classification. Note that these are a subset of
+the [CEM](../methods/CEM.ipynb) method which is also used to
+construct [pertinent negatives/counterfactuals](#4-counterfactual-instances).
+
+```{image} images/pp_mnist.png
+:align: center
+:alt: Pertinent postive of an MNIST digit
+:width: 450px
+```
+
+Given an instance $x$ we use gradient descent to find a $\delta$ that minimizes the following loss:
+
+$$ L = c\cdot L_{pred}(\delta) + \beta L_{1}(\delta, x) + L_{2}^{2}(\delta, x) + \gamma \|\delta - AE(\delta)\|^{2}_{2}
+$$
+
+$AE$ is an [autoencoder](https://en.wikipedia.org/wiki/Autoencoder) generated from the training data. If $\delta$ strays
+from the original data distribution, the autoencoder loss will increase as it will no longer reconstruct $\delta$ well.
+Thus, we ensure that $\delta$ remains close to the original dataset distribution.
+
+Note that $\delta$ is constrained to only "take away" features from the instance $x$. There is a slightly subtle point
+here: removing features from an instance requires correctly defining non-informative feature values. For
+the [MNIST digits](http://yann.lecun.com/exdb/mnist/), it's reasonable to assume that the black background behind each
+digit represents an absence of information. Similarly, in the case of colour images, you might take the median pixel
+value to convey no information, and moving away from this value adds information. For numerical tabular data we can use
+the feature mean. In general, having to choose a non-informative value for each feature is non-trivial and domain
```

Comment:
I would remove the previous 2 sentences as both examples (median pixel / feature mean) are also purely dependent on domain knowledge which is the key takeaway.